features/bootstrap/ApplicationContext.php

```diff
@@ -1,23 +1,21 @@
 <?php
 
-use Behat\Behat\Tester\Exception\PendingException;
 use Behat\Behat\Context\Context;
 use Behat\Gherkin\Node\PyStringNode;
-use Behat\Gherkin\Node\TableNode;
 use Fake\Prompter;
 use Fake\ReRunner;
-use Matcher\ApplicationOutputMatcher;
-use Matcher\ValidJUnitXmlMatcher;
 use PhpSpec\Console\Application;
 use PhpSpec\Loader\StreamWrapper;
-use PhpSpec\Matcher\MatchersProviderInterface;
 use Symfony\Component\Console\Tester\ApplicationTester;
+use PhpSpec\Exception\Example\FailureException;
 
 /**
  * Defines application features from the specific context.
  */
-class ApplicationContext implements Context, MatchersProviderInterface
+class ApplicationContext implements Context
 {
+    const JUNIT_XSD_PATH = '/src/PhpSpec/Resources/schema/junit.xsd';
```

Comment:
I think it adds to clarity to have this here - it's a shame it can't yet contain the full path (unless `.` is now ok in constants)